src/language/dafnyInstallation.ts

```diff
@@ -120,30 +136,41 @@ export async function getOrComputeLanguageServerRuntimePath(context: ExtensionCo
 
 async function cloneAllNecessaryDlls(configuredPath: string): Promise<string> {
   const dlsName = 'DafnyLanguageServer';
-  const dcName = 'DafnyCore';
   const dll = '.dll';
+  const runtimeconfigjson = '.runtimeconfig.json';
+  const depsjson = '.deps.json';
   const dls = dlsName + dll;
   if(configuredPath.endsWith(dls)) {
     const installationDir = path.dirname(configuredPath);
+    // copy all the files from installationDir to a new folder installationDir/vscode
+    const vscodeDir = path.join(installationDir, 'vscode');
+    // If the folder does not exist, create it
+    if(!await existsAsync(vscodeDir)) {
+      await mkdirAsync(vscodeDir);
+    }
+    // Copy all the files from installationDir to vscodeDir
+    const files = await fs.promises.readdir(installationDir);
+    for(const file of files) {
+      if(!(file.endsWith(dll)
+      || file.endsWith(runtimeconfigjson)
+      || file.endsWith(depsjson)
+      || file.endsWith('.pdb')
+      || file === 'z3')) {
+        continue;
+      }
+      // If it's a directory, we use the function above
+      if((await fs.promises.stat(path.join(installationDir, file)))
+        .isDirectory()) {
+        await copyDir(path.join(installationDir, file),
+          path.join(vscodeDir, file));
```

Comment:
nit
```suggestion
        await copyDir(path.join(installationDir, file), path.join(vscodeDir, file));
```

Reply:
Alternatively, if you prefer multiline, I like to place all arguments on a separate line:
```suggestion
        await copyDir(
          path.join(installationDir, file),
          path.join(vscodeDir, file)
        );
```
Similar to having a nested block of code.